float32 metr
uint32 position_x
uint32 position_y

